example/home/test/test_general.py

```diff
@@ -213,4 +216,30 @@ def test_mutation(self):
         # Now we ensure that AuthorPage-specific fields are well returned
         self.assertIn("name", data["author"])
 
+        # Finally, we ensure that data passed in the first place to the query are indeed
+        # returned after the author has been saved to the database.
         self.assertEqual(data["author"]["name"], self.name)
+        self.assertEqual(data["author"]["slug"], self.slug)
+
+
+class TestRegisterSubscription(BaseGrappleTest):
```

Comment:
because the channels implementation has different requirements (Django < 3.x), I would say move this to its own test and only call in https://github.com/GrappleGQL/wagtail-grapple/blob/main/.github/workflows/ci.yml#L52-L65, which annoyingly has a missing step 🤦🏼  (https://github.com/GrappleGQL/wagtail-grapple/blob/main/.github/workflows/ci.yml#L45-L50)

Reply:
https://github.com/GrappleGQL/wagtail-grapple/pull/174/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR51
I added the missing step here 

https://github.com/GrappleGQL/wagtail-grapple/pull/174/files#diff-b2d9000ea2939f79f0b93f509f0fcbf9cf7d88604d40b8ee1705a699bbe3040e
And a dedicated test for subscriptions here